ragtime.core/test/ragtime/repl_test.clj

```diff
@@ -38,7 +38,7 @@
 
 (deftest test-custom-reporter
   (let [database (in-memory-db)
-        config   {:datastore database :migrations migrations :reporter prn}]
+        config   {:datastore database :migrations migrations :reporter (fn [ds op id] (prn op id))}]
```

Comment:
You need to have some way of testing that the data store has been passed to the reporter. Perhaps implement `print-method` for `ragtime.core-test.InMemoryDB` so that you can test the output? Or else have the reporter write to an atom instead of STDOUT?